doc/examples/parallel_planning/parallel_planning_tutorial.rst

```diff
@@ -0,0 +1,162 @@
+How to run multiple planning pipelines in parallel with MoveItCpp?
+==================================================================
+
+MoveItCpp offers an API that allows:
+
+1. Running multiple planning pipelines in parallel with different planners
+2. Defining a custom stopping criterion that terminates the pipelines that haven't found a solution
+3. Defining a custom function to select the most suitable solutions
+
+Using multiple pipelines can be beneficial for example if:
+
+- It is unsure which planner will produce the best solution for a given planning problem
```

Comment:
"It" is a bit ambiguous here. On first read, I wasn't sure what "It" is referring to.

Reply:
You could say something like "The planner that will produce the best solution is not known a priori"